src/connection.rs

```diff
@@ -216,7 +207,7 @@ pub fn get_connections(dbus_manager: &Rc<DBusNetworkManager>) -> Result<Vec<Conn
 
 
 pub fn get_active_connections(dbus_manager: &Rc<DBusNetworkManager>)
-                              -> Result<Vec<Connection>, String> {
+    -> Result<Vec<Connection>, String> {
```

Comment:
Not sure I'm a fan of this one, although at least the arrow makes it easier to differentiate return type from the body.